cli.js

```diff
@@ -67,6 +67,8 @@ lighthouse({
   } else {
     prettyPrint(results);
   }
+}).catch(function () {
+  process.exit();
```

Comment:
because of throwing err inside .catch that make a new reject promise and then determine next promise handler. but it will be `unhandled rejection` that make the process to being hang. so that if you want to throw the error with process exit we use `process.on("unhandledRejection");

```
lighthouse().then(results => {
}).catch(function(err) {
  throw err;
  process.exit(1);
});

process.on("unhandledRejection", err => {
  throw err;
});
```